README.md

```diff
@@ -1 +1,8 @@
-# jest-assertions-workshop
\ No newline at end of file
+# jest-assertions-workshop
```

Comment:
Can we have a meaningful name for the workshop?

Reply:
Sure! 😄